internal/config/templates/k8s_conf_deploy.sh.tpl

```diff
@@ -8,30 +8,28 @@ declare -A hosts
 hosts[{{ .Hostname }}]={{ .Type }}
 {{- end }}
 
-HOSTNAME=$(cat /etc/hostname)
-if [ ! "$HOSTNAME" ]; then
-    HOSTNAME=$(cat /proc/sys/kernel/hostname)
-fi
-
-NODETYPE="${hosts[$HOSTNAME]:-server}"
+HOSTNAME=$(</proc/sys/kernel/hostname)
```

Comment:
The hostname seems to be always defined in `/proc/sys`.

Reply:
is this actually fixing something or was this unrelated cleanup? I do agree that /proc/sys/kernel/hostname should be preferred, but I just feel nervous about doing that change without extensive testing. 

if it was unrelated can we just please drop it and push it to another PR? I'm happy with all the other changes.